### ISSUE TITLE
Running onvm-snort on docker container

### ISSUE THREAD

**Issue by madhura-a** [sdnfv/onvm-snort#11]
Is it possible to run onvm-snort on docker containers using the docker.sh script present in openNetVM?

**Comment by twood02** [comment]
I don't think we have tested this, but I don't know of a reason why it wouldn't work.

If you are using docker, we suggest you use the version in this pull request  https://github.com/sdnfv/openNetVM/pull/202
It isn't merged yet, but you can pull the code from here: https://github.com/kevindweb/openNetVM/tree/docker_ubuntu18_update
and dockerhub now has an updated docker image.
